packages/tooltip/src/getTooltipTriggerHandlers.ts

```diff
@@ -0,0 +1,31 @@
+import { HTMLProps } from 'react';
+
+/**
+ * Isolates tooltip trigger methods from all props (i.e. `rest`)
+ * in order to pass them as a unit into another component.
+ * This is useful when you need  tooltips to work with the component
+ * and need to spread rest on an element other than the component root.
+ *
+ * e.x.
+ * ```ts
+ * cosnt tooltipHandlers = getTooltipTriggerHandlers(rest)
+ * <label {...tooltipHandlers}>
+ *  <input {...rest} />
+ * </label>
+ * ```
+ */
+export function getTooltipTriggerHandlers<T = any>({
```

Comment:
I figures we may need this logic again elsewhere, so I added this to `Tooltip` instead of having a one-off solution in Radiobox

Reply:
Why do we need this? It looks like it's just putting all the props in an object and returning it?